### ISSUE TITLE
[FEA] better url settings docs

### ISSUE THREAD

**Issue by lmeyerov** [graphistry/pygraphistry#537]
**Is your feature request related to a problem? Please describe.**

A pygraphistry user couldn't find, with decent effort, how to set edge opacity to 0.1 

ChatGPT hallucinated the wrong answer too, and took much cajoling to find:  https://chat.openai.com/share/c22a10cf-b0c3-40d8-96ab-715d8b549da9

pyg docs should have been easier here, and in a way github/docs scrapers could have picked up

Right now, `.settings()` doesn't say what valid `url_params={...}` can be (https://github.com/graphistry/pygraphistry/blob/e74123e7d82d2797d9802b0fa3011ad517f251e5/graphistry/PlotterBase.py#L1182), particularly wrt https://hub.graphistry.com/docs/api/1/rest/url/#urloptions 

**Describe the solution you'd like**

* `help(g.settings)` would have pointed out the `url_params={...}` options from https://github.com/graphistry/pygraphistry/blob/e74123e7d82d2797d9802b0fa3011ad517f251e5/graphistry/PlotterBase.py#L1182 
* overall pygraphistry docstrs would have this => search engines / LLMs, so probably inline somehow, vs outside link
* demo notebook of most features: https://github.com/graphistry/pygraphistry/tree/master/demos/more_examples/graphistry_features
  * basically a python analogue of https://hub.graphistry.com/static/js-docs/index.html?path=/docs/graphistry-react-style--empty

As part of the impl, maybe an idea is creating an openai / pydantic out of this, so it is machine processed? https://github.com/graphistry/graphistry/blob/eab8648d4ee9c9f3535baeb1aa5ac0904e6a3294/apps/core/viz/server/clientParamSafelist.js#L141 ?
